src/scripts/utility-commands/time-and-date.js

```diff
@@ -0,0 +1,52 @@
+const { DateTime } = require("luxon");
+
+function convertBetweenTimezones(message) {
+    let text = message.content ?? "";
+    if (text.length < 10) {
+        return;
+    }
+    text = text.substring(9).trim();
+
+    const dateTime = getDateTimeFromMessage(text);
+    if (dateTime === null) {
+        message.reply("please provide a valid datetime in the format YYYY/MM/DD HH:mm");
+        return;
+    }
+
+    const timezones = getTimeZonesFromMessage(text);
+    if (timezones === null || timezones.length === 0) {
+        message.reply("please provide at least one valid timezone to convert to. \n\nFor a list of valid timezones see the *'TZ database name'* column in this table here: <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List>");
+        return;
+    }
```

Comment:
Awesome! I really like the functionality of it for now.
We may want to revisit this later and ask ourselves "how easy-to-use is this for users?"

Perhaps including one of your example inputs in the message.reply() here?